.github/workflows/test-pr.yml

```diff
@@ -180,3 +180,49 @@ jobs:
         if: always()
         run: |
           docker stop --time=0 ${CONTAINER_NAME}
+
+  nix:
+    needs: code-quality-checks
+    name: 'Nix Tests'
+    strategy:
+      fail-fast: false
+      matrix:
+        runner: [ubuntu-latest, macos-13]
+    runs-on: ${{ matrix.runner }}
+    timeout-minutes: 60
+    steps:
+      - name: 'Check out code'
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: 'Install Foundry'
+        uses: foundry-rs/foundry-toolchain@v1
```

Comment:
Installing Foundry separately is necessary in the current setup.

Reply:
If you use `nix build .#kontrol.solc_0_8_13`, foundry and solc will be added to the kontrol executable's PATH

Reply:
That did the trick, thanks!